src/TestFramework/Coverage/JUnit/JUnitTestExecutionInfoAdder.php

```diff
@@ -103,7 +103,7 @@ private function createCompleteTestLocation(TestLocation $test): TestLocation
         return new TestLocation(
             $test->getMethod(),
             $testFileData->path,
-            $testFileData->time,
+            $test->getExecutionTime() ?? $testFileData->time,
```

Comment:
There are per-test timings in the JUnit report I had lying around, so in theory it should be possible to get a more accurate numbers.

Reply:
could you please explain the idea of this PR? Sorry, without debugging I think I don't get it

Reply:
The idea: https://github.com/infection/infection/issues/1965#issuecomment-2119078924